charts/nebraska/templates/deployment.yaml

```
@@ -81,11 +81,39 @@ spec:
             {{- if .Values.config.auth.mode }}
             - "-auth-mode={{ .Values.config.auth.mode }}"
             {{- end }}
-            {{- if and .Values.config.auth.github.readWriteTeams (not (eq .Values.config.auth.mode "noop")) }}
+            {{- if eq .Values.config.auth.mode "github" }}
+              {{- if .Values.config.auth.github.readWriteTeams }}
             - "-gh-rw-teams={{ .Values.config.auth.github.readWriteTeams }}"
-            {{- end }}
-            {{- if and .Values.config.auth.github.readOnlyTeams (not (eq .Values.config.auth.mode "noop")) }}
+              {{- end }}
+              {{- if .Values.config.auth.github.readOnlyTeams }}
             - "-gh-ro-teams={{ .Values.config.auth.github.readOnlyTeams }}"
+              {{- end }}
+            {{- end }}
+            {{- if eq .Values.config.auth.mode "oidc" }}
+              {{- if .Values.config.auth.oidc.issuerURL }}
+            - "-oidc-issuer-url={{ .Values.config.auth.oidc.issuerURL }}"
```

Maybe it would make more sense to wrap every command line option with the keyword `with` instead of `if`. Then we do not need to write the values path twice. WDYT? Applies also for the following lines.
```suggestion
              {{- with .Values.config.auth.oidc.issuerURL }}
            - "-oidc-issuer-url={{ . }}"
```

---

I have made this change for the OIDC values alone for now and skipped it for the GitHub values as it will be removed eventually.